source/content/guides/local-development/04-jenkins.md

```diff
@@ -13,6 +13,7 @@ layout: guide
 showtoc: true
```

Comment:
@stovak , contains multiple references to D8. Should it say “prior version” instead?  See preview link: https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/local-development/jenkins